firstvoices/backend/serializers/join_request_serializers.py

```diff
@@ -13,7 +16,7 @@
 
 
 class JoinRequestDetailSerializer(BaseSiteContentSerializer):
-    url = SiteHyperlinkedIdentityField(view_name="api:join-request-detail")
+    url = SiteHyperlinkedIdentityField(view_name="api:joinrequest-detail")
```

Comment:
I changed this only because if django ever tried to auto-generate a url for a join request, it would use the non-hyphenated version (just lowercases the model name)